mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -68,6 +84,7 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
   def setLabelCol(value: String): this.type = set(labelCol, value)
 
   setDefault(metricName -> "f1")
+  setDefault(labelValue -> -1.0)
```

Comment:
I think it is cleaner to not set a default. You can check whether the param is set using `isSet`.